app/data_client.go

```diff
@@ -183,6 +183,14 @@ type DatabaseConnReturn struct {
 	HasDatabaseUser bool
 }
 
+// LatestTabularDataReturn represents the response returned by GetLatestTabularData. It contains the most recently captured data
+// payload, the time it was captured, and the time it was synced.
+type LatestTabularDataReturn struct {
+	TimeCaptured time.Time
+	TimeSynced   time.Time
+	Payload      *structpb.Struct
```

Comment:
We try to refrain from exposing proto types like this struct. I would make `Payload` a `[]*map[string]interface{}` type. Then define a `latestTabularDataReturnFromProto` function where you use `resp.Payload.AsMap()`.

Reply:
Roger that, changed!